src/components/ButtonPanel.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Button from './Button';
+
+class ButtonPanel extends React.Component {
+    
+        
+    render() {
+    <Button />
+        return(
```

Comment:
- [ ] Kindly remove the button component called here and implement it as explained in the comment above